apps/web/src/pages/SearchRequests/RequestPage/components/RequestForm.tsx

```diff
@@ -337,6 +338,18 @@ export const RequestForm: React.FunctionComponent<RequestFormProps> = ({
                   "Blurb before additional comments textarea in the request form.",
               })}
             </p>
+            {candidateCount === 0 ? (
+              <p data-h2-margin="base(x.5, 0, 0, 0)">
+                {intl.formatMessage({
+                  defaultMessage:
+                    "If you are submitting a form that had zero estimated candidates, let us know moe about this request in the comments.",
```

Comment:
"let us know moe"? 😛 
probably need to check the translation too

Reply:
weird, i totally fixed that. must have forgot to commit lol

[fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5601/commits/5bdbb8df80813a5e9e43b6d6123d8cf62e795556)